templates/event/event_show.html.twig

```diff
@@ -1,118 +1,128 @@
 {% extends 'base.html.twig' %}
 
-{% block heading %}
-    {% if event %}
-        {{ event.name }}
-    {% else %}
-        Please specify valid event.
-    {% endif %}
+{% block page_heading %}
+    <div class="row justify-content-center" >
+        <div class="col-11">
+            <h2 class="sub-header text-center brand-txt">
+                {% block heading %}
+                    {% if event %}
+                        {{ event.name }}
+                    {% else %}
+                        Событие не найдено.
+                    {% endif %}
+                {% endblock %}
+            </h2>
+        </div>
+    </div>
 {% endblock %}
 
 {% block content %}
     {% if event %}
+        <div class="row justify-content-center">
+            <div class="col-11">
+                {%  if event.picture %}
+                    <img class="img-fluid event-img mr-3 my-2" src="{{ asset( 'images/pictures/' ~ event.picture) }}" alt="{{ event.name }}" align="left">
+                {% else %}
+                    <img class="img-fluid event-img mr-3 my-2"  src="{{ asset("images/rolendar-1.jpg") }}" alt="{{ event.name }}" align="left">
+                {% endif %}
 
-        {%  if event.picture %}
-            <img class="img-fluid event-img mr-3 my-2" src="{{ asset( 'images/pictures/' ~ event.picture) }}" alt="{{ event.name }}" align="left">
-        {% else %}
-            <img class="img-fluid event-img mr-3 my-2"  src="{{ asset("images/rolendar-1.jpg") }}" alt="{{ event.name }}" align="left">
-        {% endif %}
-
-        {% if event.status == constant('STATUS_CANCELLED', event) %}
-            <h4 class="p-2">
-                <span class="badge badge-danger">Отменено!</span>
-            </h4>
-        {% endif %}
-
-        <h4 class="text-muted pl-1">
-            {{ event.startDate | date("d/m/Y") }}
-            {% if event.startDate != event.endDate %}
-                 - {{ event.endDate | date("d/m/Y") }}
-            {% endif %}
-        </h4>
-        <h4 class="text-muted pl-1">
-            {% if event.location %}
-                {{ event.location }},
-            {% endif %}
-            {{ event.region.name }}
-        </h4>
-
-        {% if event.type %}
-            <p class="pl-1"> <strong> Формат:</strong> {{ event.type.name}} </p>
-        {% endif %}
-
-        {% if event.genre %}
-            <p class="pl-1"> <strong> Жанр:</strong> {{ event.genre.name }} </p>
-        {% endif %}
+                {% if event.status == constant('STATUS_CANCELLED', event) %}
+                    <h4 class="p-2">
```

Comment:
@daria-sieroshtan 
I would do this with something like `bade-danger`

Reply:
It is done exactly this way